encoders/numeric/CompressionVaeEncoder/__init__.py

```diff
@@ -15,18 +15,22 @@
 
 class CompressionVaeEncoder(TFDevice, BaseNumericEncoder):
     """
-    :class:`CompressionVaeEncoder` is a dimensionality reduction tool based on the idea of
-    Variational Autoencoders. It encodes data from an ndarray in size `B x T` into an ndarray in size `B x D`.
+    :class:`CompressionVaeEncoder` is a dimensionality reduction tool.
 
-    Full code and documentation can be found here: https://github.com/maxfrenzel/CompressionVAE..
+    It is based on the idea of Variational Autoencoders.
+    It encodes data from an ndarray in size `B x T` into an ndarray
+    in size `B x D`. Where `B` is the batch's size and `T` and `D`
+    are the dimensions pre (`T`) and after (`D`)the compression.
+
+    Full code and documentation can be found here:
+    https://github.com/maxfrenzel/CompressionVAE.
+
+    :param model_path: Path to the pretrained model
```

Comment:
can you add args and kwargs absed on our guide: https://docs.jina.ai/chapters/docstring/docstring.html#args-and-kwargs